auditbeat/scripts/docs_collector.py

```diff
@@ -133,28 +148,48 @@ def collect(beat_name):
             module_file += module_includes
 
         # Write module docs
-        with open(os.path.abspath("docs") + "/modules/" + module + ".asciidoc", 'w') as f:
+        with open(os.path.join(module_docs_path(module_dir), "modules", module + ".asciidoc"), 'w') as f:
             f.write(module_file)
 
     module_list_output = generated_note
     for m, title in sorted(six.iteritems(modules_list)):
+        if m == "system":
+            # This is a very crude way of excluding references to X-Pack docs from
+            # the build until the docs build is updated with the new folder.
+            # TODO: Remove at the earliest possible time.
```

Comment:
I don't have a good choice, unfortunately. The `x-pack/auditbeat/docs` directory has to exist before it can be added to the docs build, but only after it has been added can references to it be resolved. So the plan is check it in here without docs references to it, then add it to the docs build, then re-enable references.